app/main.py

```diff
@@ -1,10 +1,10 @@
 def format_linter_error(error: dict) -> dict:
     return {
-        it: [
```

Comment:
Couldn't stop thinking I had to use dict comprehension and complicated everything too much, I now realize it)